sync/sync-impl/src/main/res/values/donottranslate.xml

```diff
@@ -47,4 +49,9 @@
     <string name="sync_recovery_code_primary_button">Save Recovery PDF</string>
     <string name="sync_recovery_code_secondary_button">Copy Code</string>
     <string name="sync_recovery_code_skip_button">Not Now</string>
+
+    <string name="sync_connected_device_title">Device Synced!</string>
+    <string name="sync_connected_device_hint">Your bookmarks are now syncing with this device.</string>
```

Comment:
note to bring this up for discussion, we are not really syncing bookmarks when this text is displayed.

Reply:
we can review during ship review, but I don't think is incorrect. It can be read as: we are syncing your bookmarks now (which may be correct since it's expected to happen after login), or FYI this device syncs bookmarks from now on (which is also true).